sdk/search/Azure.AI.Search/src/Azure.AI.Search.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure.AI.Search client library</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <Description>
+      Query and manage Azure Cognitive Search indexes.
+    </Description>
+    <PackageTags>Azure Cognitive Search;Azure Search;Search;Cognitive;Search Engine;Azure</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    
+    <!-- We don't have to worry about Breaking Changes until GA -->
+    <EnableApiCompat>false</EnableApiCompat>
+
+    <!-- *ClientOptions Analyzer blocked by https://github.com/Azure/azure-sdk-tools/issues/127 -->
+    <NoWarn>$(NoWarn);AZC0007;</NoWarn>
+
+    <!-- Ignore generated warnings until https://github.com/Azure/autorest.csharp/issues/206 is fixed -->
+    <NoWarn>
+      $(NoWarn);
+      AZC0001;
+      CA1056;
+      CA1507;
+      CA1717;
+      CA1812;
+      CS1591;
+      SA1649;
+    </NoWarn>
+  </PropertyGroup>
+
+  <!-- Pull in Shared Source from Azure.Core -->
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)ArrayBufferWriter.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" Link="Shared\Core\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)ClientDiagnostics.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)ContentTypeUtilities.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)DiagnosticScope.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)DiagnosticScopeFactory.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)ForwardsClientCallsAttribute.cs" Link="Shared\Core\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)HttpMessageSanitizer.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)TaskExtensions.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+  </ItemGroup>
+
+  <!-- Project and Package references -->
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
+  <ItemGroup>
+    <PackageReference Include="System.Text.Json" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
nit: rem this item group

Reply:
VS will automatically remove it when I'm able to move my `CodeGenSchema` renames to that folder.  :)